apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -94,15 +95,28 @@ export function StudentAssessmentInstance({
 )) {
   let savedAnswers = 0;
   let suspendedSavedAnswers = 0;
+
+  // Check for mixed real-time grading scenarios
+  const someQuestionsAllowRealTimeGrading = instance_question_rows.some(
+    (q) => q.allow_real_time_grading,
+  );
+  const someQuestionsForbidRealTimeGrading = instance_question_rows.some(
```

Comment:
Will this pick up `null` by accident, or is that on purpose?

Reply:
This was written with the eventual state of this code in mind: `assessment_questions.allow_real_time_grading` will have a `NOT NULL` constraint.

Reply:
I see, thanks.

Reply:
I added a note about nullability in ae9b26a. That'll remind us to double-check this before finally merging. I'll resolve this discussion for now.